sempyro/hri_dcat/hri_distribution.py

```diff
@@ -89,6 +106,20 @@ class HRIDistribution(DCATDistribution):
             "rdf_type": "uri"
         }
     )
+    rights: AnyHttpUrl = Field(
+        description="Information about rights held in and over the distribution.",
+        json_schema_extra={
+            "rdf_term": DCTERMS.rights,
+            "rdf_type": "uri"
+        }
+    )
+    byte_size: Union[int, LiteralField] = Field(
+        description="The size of a distribution in bytes.",
+        json_schema_extra={
+            "rdf_term": DCAT.byteSize,
+            "rdf_type": "xsd:nonNegativeInteger"
+        }
+    )
```

Comment:
We're having some issues with the `xsd:nonNegativeInteger` types in the FDP client. We'll need to make sure this is in sync with the choices down the line.